sensors/lidar/dim-2d/dim-2d_test.go

```diff
@@ -156,7 +156,7 @@ func TestGetTimedData(t *testing.T) {
 		pcTime, pc, err := dim2d.GetTimedData(ctx, goodLidar)
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, pc, test.ShouldNotBeNil)
-		test.That(t, pcTime.After(beforeReading), test.ShouldBeTrue)
+		test.That(t, pcTime.Before(beforeReading), test.ShouldBeFalse)
```

Comment:
This was needed to make these tests reliable. It appears as if it is possible for `pcTime` and `beforeReading` to be equal to each other. This function just needs to not return a pcTime which is from before the function was called, so this test now reflects that.